File: src/main/java-graphql/io/github/graphql/java/resolver/RegistryPackageSearchResolver.java
package io.github.graphql.java.resolver;

import io.github.graphql.java.model.*;

/**
 * Resolver for RegistryPackageSearch
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public interface RegistryPackageSearchResolver {

    @javax.validation.constraints.NotNull
    RegistryPackageConnectionTO registryPackagesForQuery(RegistryPackageSearchTO registryPackageSearchTO, String after, String before, Integer first, Integer last, RegistryPackageTypeTO packageType, String query) throws Exception;

}

File: src/main/kotlin-graphql/io/github/graphql/kotlin/resolver/RegistryPackageSearchResolver.kt
package io.github.graphql.kotlin.resolver

import io.github.graphql.kotlin.model.*

/**
 * Resolver for RegistryPackageSearch
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-07T19:48:14+0800"
)
interface RegistryPackageSearchResolver {

    @Throws(Exception::class)
    fun registryPackagesForQuery(registryPackageSearchTO: RegistryPackageSearchTO, after: String?, before: String?, first: Int?, last: Int?, packageType: RegistryPackageTypeTO?, query: String?): RegistryPackageConnectionTO

}
